packages/@ama-sdk/showcase-sdk/readme.md

```diff
@@ -1,6 +1,9 @@
 # Showcase SDK
 
-### Based on Swagger spec 1.0.17
+>Note:
+This package is intended for **testing purposes only**. It is not suitable for production use.
```

Comment:
Maybe we could set a `deprecated` message in the package.json so it will display a big warning while looking at the package on npmjs

Reply:
did not know about this. updated :)